fog/report/server/src/service.rs

```diff
@@ -103,8 +103,9 @@ impl<R: ReportDb + Clone + Send + Sync> Service<R> {
         let signature = self
             .materials
             .signing_keypair
-            .sign_reports(&reports[..])?
-            .as_bytes()
+            .sign_reports(&reports[..])
+            .map_err(|_| Error::Signature)?
```

Comment:
Similar: `impl From<SignatureError> for Error`

Reply:
Done. (See above comment).